### PR TITLE
Custom name for objects in a collection

### PR DESCRIPTION
**Pull request by ebertti**
Just another attribute inside of Meta class of Resource

### CODE REVIEW

File: tastypie/resources.py [hunk 1]
```diff
@@ -80,6 +80,7 @@ class ResourceOptions(object):
     include_absolute_url = False
     always_return_data = False
     collection_name = 'objects'
+    object_name = None
```
Comment (SeanHayes) [inline tastypie/resources.py]: This is never set anywhere.

**Comment by SeanHayes** [comment]
We'll need to know what the use case is.

**Comment by ebertti** [comment]
Better customization and integrations with legace systems

**Comment by SeanHayes** [comment]
Could you provide an example? What are you trying to do? You can already define different field names on your Resources than you have on your models. The case for adding a new attribute which we'll have to support, and which will only be used by a single serializer, doesn't seem strong enough to me.

**Comment by SeanHayes** [comment]
Wait, I just saw the Gist, I get it now. I'll try to think if there's a way we can do this.

**Comment by SeanHayes** [comment]
#1380

**Pull request closed without merging**